This fix removes python in the usage string from the help.

--- sqlmap/lib/parse/cmdline.py.Orig	2021-04-15 23:16:45.000000000 +0300
+++ sqlmap/lib/parse/cmdline.py		2021-04-15 23:19:20.000000000 +0300
@@ -107,7 +107,7 @@
     # Reference: https://stackoverflow.com/a/4012683 (Note: previously used "...sys.getfilesystemencoding() or UNICODE_ENCODING")
     _ = getUnicode(os.path.basename(argv[0]), encoding=sys.stdin.encoding)
 
-    usage = "%s%s [options]" % ("%s " % os.path.basename(sys.executable) if not IS_WIN else "", "\"%s\"" % _ if " " in _ else _)
+    usage = "sqlmap [options]"
     parser = ArgumentParser(usage=usage)
 
     try:
